bundler/spec/realworld/edgecases_spec.rb

```diff
@@ -196,21 +196,6 @@ def rubygems_version(name, requirement)
     expect(lockfile).to include(rubygems_version("paperclip", "~> 5.1.0"))
   end
 
-  # https://github.com/rubygems/bundler/issues/1500
```

Comment:
Any details why this is removed?

Reply:
I explained it in the commit message. This is the only realworld spec that downloads two real gems and installs them through  `gem install`. It's the only spec that fails when run offline after my changes.

I thought of adapting the specs to run offline, but then I started digging into git history and my conclusion was that this spec is not covering us from any regressions, since I completely removed all code in bundler regarding rubygems plugin installation (which is what the spec is supposed to be testing) and the spec still passed.

So, since the spec didn't seem to be doing any good, I decided to just kill it.

Reply:
Damn, sorry. I went thru the PR description few times since I was sure you'll not remove anything without explaining. But I forgot to check commit messages.

Thanks for the explanation. :+1:

Reply:
No problem at all!